app/i18n/locales/en.json

```diff
@@ -257,10 +257,10 @@
   "Forward_to_user": "Forward to user",
   "Full_table": "Click to see full table",
   "Generate_New_Link": "Generate New Link",
-  "Has_joined_the_channel": "has joined the channel",
```

Comment:
Considering that "Has_joined_the_channel" will not use anymore, can you delete the keys and strings from other language files?